ruma-common/Cargo.toml

```diff
@@ -24,3 +24,4 @@ matches = "0.1.8"
 
 [features]
 unstable-exhaustive-types = []
+unstable-synapse-quirks = []
```

Comment:
Ah, this isn't caught by CI but we really should have a newline before EOF everywhere.

Reply:
You can enforce that with `.gitattributes` in the root of the repository with `* text=auto` as its content.